contracts/validator-manager/ERC20TokenStakingManager.sol

```diff
@@ -119,6 +119,13 @@ contract ERC20TokenStakingManager is
         return _initializeDelegatorRegistration(validationID, _msgSender(), delegationAmount);
     }
 
+    /**
+     * @notice Returns the ERC20 token being staked
+     */
+    function erc20() external view returns (IERC20Mintable) {
```

Comment:
I added this to allow the deployment utility to return an `IPoSValidatorManager` object, and recover the `ERC20` instance after the fact. It also seems reasonable that use cases will emerge that will want to access the staked asset programatically.